src/pages/content-edit/head.css

```diff
@@ -1,36 +1,22 @@
 @value pure: "../../_pure.css";
-@value   icons_css: "../../icons.css";
-@value  layout_css: "../layout/layout.css";
+@value icons_css: "../../icons.css";
+@value layout_css: "../layout/layout.css";
 @value buttons_css: "../../buttons.css";
 @value header: "../layout/header.css";
 
-.head {
+.contentHead {
```

Comment:
I like it, prefer `.contentHd` and `.contentBd` though.

Naming things is a PITA.